s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -405,3 +456,79 @@ func (t *TidbClient) UpdateUsages(ctx context.Context, usages map[string]int64,
 	}
 	return nil
 }
+
+func buildSql(ctx context.Context, filter map[string]string, sqltxt string) (string, []interface{}, error) {
+	const MaxObjectList = 10000
```

Comment:
please move this to an outside configuration file or such place

Reply:
Done